servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultHttpLoadBalancerFactory.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright © 2020 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.http.netty;
+
+import io.servicetalk.client.api.ConnectionFactory;
+import io.servicetalk.client.api.LoadBalancer;
+import io.servicetalk.client.api.LoadBalancerFactory;
+import io.servicetalk.client.api.ServiceDiscovererEvent;
+import io.servicetalk.concurrent.api.Publisher;
+import io.servicetalk.http.api.FilterableStreamingHttpConnection;
+import io.servicetalk.http.api.FilterableStreamingHttpLoadBalancedConnection;
+import io.servicetalk.http.api.HttpExecutionStrategy;
+import io.servicetalk.http.api.HttpExecutionStrategyInfluencer;
+import io.servicetalk.http.api.HttpLoadBalancerFactory;
+import io.servicetalk.loadbalancer.RoundRobinLoadBalancer;
+import io.servicetalk.loadbalancer.RoundRobinLoadBalancer.RoundRobinLoadBalancerFactory;
+
+import static io.servicetalk.http.api.HttpExecutionStrategyInfluencer.defaultStreamingInfluencer;
+import static io.servicetalk.loadbalancer.RoundRobinLoadBalancer.newRoundRobinFactory;
+
+/**
+ * Default implementation of {@link HttpLoadBalancerFactory}.
+ *
+ * @param <ResolvedAddress> The type of address after resolution.
+ */
+public final class DefaultHttpLoadBalancerFactory<ResolvedAddress>
+        implements HttpLoadBalancerFactory<ResolvedAddress>, HttpExecutionStrategyInfluencer {
+    private final Config<ResolvedAddress> config;
+
+    private DefaultHttpLoadBalancerFactory(final Config<ResolvedAddress> config) {
+        this.config = config;
+    }
+
+    @Override
+    public LoadBalancer<? extends FilterableStreamingHttpLoadBalancedConnection> newLoadBalancer(
+            final Publisher<? extends ServiceDiscovererEvent<ResolvedAddress>> eventPublisher,
+            final ConnectionFactory<ResolvedAddress, ? extends FilterableStreamingHttpLoadBalancedConnection> cf) {
+        return config.rawFactory.newLoadBalancer(eventPublisher, cf);
+    }
+
+    @Override
+    public FilterableStreamingHttpLoadBalancedConnection toLoadBalancedConnection(
+            final FilterableStreamingHttpConnection connection) {
+        return new AbstractFilterableStreamingHttpLoadBalancedConnection(connection) {
+            @Override
+            public float score() {
+                return 1;
```

Comment:
Looking at #996, should it be `Integer.MAX_VALUE`?

Reply:
Should we provide any way for users to configure this value? For example, consider adding a method for the builder that will take `Function<FilterableStreamingHttpConnection, FilterableStreamingHttpLoadBalancedConnection>`.

Reply:
>> Looking at #996, should it be Integer.MAX_VALUE?

Yes! I will change it here or in the other PR depending on which gets merged first.

How scores are used/specified depends on the load balancer requirements. eg: the only load balancer we have ATM does not use the scores. I will avoid adding new APIs speculating usage. Once we have something that uses the scores, we can decide the APIs. Until then, one can create their own `HttpLoadBalancerFactory` implementation.